packages/nouns-contracts/test/foundry/helpers/DeployUtils.sol

```diff
@@ -90,9 +91,24 @@ abstract contract DeployUtils is Test, DescriptorHelpers {
         return descriptorV2;
     }
 
+    function _deployAndPopulateV3() internal returns (NounsDescriptorV3) {
```

Comment:
@EricDevito I noticed there are a bunch of descriptor-related tests in solidity (foundry), and not all of them are here in your diff. The main file is `NounsDescriptorV2.t.sol`. Seems like it has relevant tests you want to make sure apply to V3 as well?